contrib/python/api/skydive/packet_injector.py

```diff
@@ -23,7 +23,7 @@ class PacketInjection(object):
     def __init__(self, uuid="", src="", dst="", srcip="", dstip="", srcmac="",
                  dstmac="", srcport="", dstport="", type="", payload="",
                  trackingid="", icmpid="", count="", interval="",
-                 increment="", starttime=""):
+                 increment="", starttime="", ttl=""):
```

Comment:
this should be 255 by default. This makes me think that count, interval, increment should be numbers

Reply:
modified all the fields